packages/flutter_tools/lib/src/compile.dart

```diff
@@ -18,6 +19,9 @@ String _dartExecutable() {
   final String engineDartSdkPath = artifacts.getArtifactPath(
     Artifact.engineDartSdkPath
   );
+  if (!fs.isDirectorySync(engineDartSdkPath)) {
+    throwToolExit('No dart sdk Flutter host engine build found at $engineDartSdkPath.', exitCode: 2);
```

Comment:
Given the nature of the message I think it would be interesting to be more explicit.
```dart
throwToolExit('No dart sdk found in the Flutter host engine build at $engineDartSdkPath.\n'
              'The Flutter host engine is required even to target other platforms', exitCode: 2);
```
What do you think?

Reply:
Sure, I updated the message taking yours as a base, modifying it slightly.

Reply:
Great